dist/tools/autotest/autotest/utils.py

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+# vim:fenc=utf-8
+#
+# Copyright © 2016 Martine Lenders <mail@martine-lenders.eu>
+#
+# Distributed under terms of the MIT license.
+
+class Borg(object):
```

Comment:
I have to credit [David Asher](http://code.activestate.com/recipes/users/70683/) for that name ;-)